frontend/src/utils/timeUtils.ts

```diff
@@ -0,0 +1,5 @@
+export const isSessionExpired = (unixTimestamp: number): boolean => {
```

Comment:
This function name `isSessionExpired` looks dependent on the feature of Supabase session.  As utility function of time, you can consider the more common name. How about`hasExceededOnNow`?

Reply:
I got it. I changed to hasExceededOnNow.

```js
export const hasExceededOnNow = (unixTimestamp: number): boolean => {
```